src/main/resources/assets/gtceu/lang/ru_ru.json

```diff
@@ -6787,5 +6787,15 @@
     "gtceu.top.energy_stored": " / %d EU",
     "gtceu.top.progress_computation": " / %s CWU",
     "item.gtceu.portable_scanner": "Портативный сканер",
-    "gtceu.multiblock.uv_fusion_reactor.description": "Термоядерный реактор модель 3 - это большая многоблочная структура использующаяся для термоядерного синтеза двух элементов в один. Может использовать UV+ энергетические разъемы. За каждый энергетический разъём, внутренний буфер увеличивается на 40млн. EU, максимум энергии для старта 640млн. EU."
+    "gtceu.multiblock.uv_fusion_reactor.description": "Термоядерный реактор модель 3 - это большая многоблочная структура использующаяся для термоядерного синтеза двух элементов в один. Может использовать UV+ энергетические разъемы. За каждый энергетический разъём, внутренний буфер увеличивается на 40млн. EU, максимум энергии для старта 640млн. EU.",
+    "config.gtceu.option.bedrockOreDistance": "bedrockOreDistance",
+    "gtceu.multiblock.ore_rig.drilled_ore_entry": " - %s",
+    "gtceu.multiblock.ore_rig.drilled_ores_list": "Руды:",
+    "gtceu.multiblock.ore_rig.ore_amount": "Скорость доб.: %s",
+    "tagprefix.door": "Дверь (%s)",
+    "tagprefix.fence": "Забор (%s)",
+    "tagprefix.fence_gate": "Калитка (%s)",
+    "tagprefix.log": "Дерево (%s)",
+    "tagprefix.slab": "Плита (%s)",
+    "tagprefix.stairs": "Ступеньки (%s)"
```

Comment:
why are these formatted as "stairs (material)" and not "material stairs"?

Reply:
Material in this case better use as noun in the nominative case, to avoid situation when we have "Золот**ая** отвертка" and "Золот**ая** меч" but last one must be "Золот**ой** меч".

Because of that in 1.12.2 and in 1.20.2 i use this format. And this solve problem with result as: Кирка (Золото) and Меч (Золото)

Reply:
👍